config/vlan.py

```diff
@@ -201,3 +201,333 @@ def del_vlan_dhcp_relay_destination(db, vid, dhcp_relay_destination_ip):
         clicommon.run_command("systemctl restart dhcp_relay", display_cmd=False)
     except SystemExit as e:
         ctx.fail("Restart service dhcp_relay failed with error {}".format(e))
+
+#
+# 'range' group ('config vlan range ...')
+#
+@vlan.group('range')
+@click.pass_context
+def vlan_range(ctx):
+    """VLAN-range related configuration tasks"""
+    pass
+
+@vlan_range.command('add')
+@click.argument('vid1', metavar='<vid1>', required=True, type=int)
+@click.argument('vid2', metavar='<vid2>', required=True, type=int)
+@click.option('-w', "--warning", is_flag=True, help='warnings are not suppressed')
+@click.pass_context
+def add_vlan_range(ctx, vid1, vid2, warning):
+    db = ctx.obj['db']
+
+    vlan_range_validate(ctx, vid1, vid2)
+
+    vid2 = vid2+1
+
+    warning_vlans_list = []
+    curr_vlan_count = 0
+    clients = db.redis_clients["CONFIG_DB"]
+    pipe = clients.pipeline()
+    for vid in range (vid1, vid2):
+        vlan = 'Vlan{}'.format(vid)
+
+        if len(db.get_entry('VLAN', vlan)) != 0:
+            if warning is True:
+                warning_vlans_list.append(vid)
+            continue
+
+        pipe.hmset('VLAN|{}'.format(vlan),  {'vlanid': vid})
+        curr_vlan_count += 1
+    pipe.execute()
+    # Log warning messages if 'warning' option is enabled
+    if warning is True:
+        if len(warning_vlans_list) != 0:
+            logging.warning('VLANs already existing: {}'.format(get_hyphenated_string(warning_vlans_list)))
+
+@vlan_range.command('del')
+@click.argument('vid1', metavar='<vid1>', required=True, type=int)
+@click.argument('vid2', metavar='<vid2>', required=True, type=int)
+@click.option('-w', "--warning", is_flag=True, help='warnings are not suppressed')
+@click.pass_context
+def del_vlan_range(ctx, vid1, vid2, warning):
+    db = ctx.obj['db']
+
+    vlan_range_validate(ctx, vid1, vid2)
+
+    vid2 = vid2+1
+
+    warning_vlans_list = []
+    warning_membership_list = []
+    warning_ip_list = []
+    clients = db.redis_clients["CONFIG_DB"]
+    pipe = clients.pipeline()
+    vlan_member_keys = db.keys('CONFIG_DB', "*VLAN_MEMBER*")
```

Comment:
suggest to use `clients.scan(...)` API

Reply:
@anilkpandey [client scan](https://redis.io/commands/scan) it use cursor iterator so it does not lock the redis db for long time. [Here](https://github.com/Azure/sonic-py-swsssdk/blob/master/src/swsssdk/configdb.py#L429) is a code that uses client scan in swsssdk.